spinn_front_end_common/interface/spinnaker.cfg

```diff
@@ -162,7 +163,8 @@ boot_connection_port_num = None
 auto_detect_bmp = False
 turn_off_machine = False
 clear_routing_tables = False
-clear_tags = False
+clear_tags = False"when there is nothing left to remove
```

Comment:
Oops?

Reply:
..................... how did that get in............... oh well. yep. oops

Reply:
fixed